packages/pieces/facebook-pages/src/lib/common/index.ts

```diff
@@ -0,0 +1,163 @@
+import { OAuth2PropertyValue, Property } from "@activepieces/pieces-framework";
+import { HttpRequest, HttpMethod, httpClient } from "@activepieces/pieces-common";
+
+export const facebookPagesCommon = {
+    baseUrl: 'https://graph.facebook.com/v17.0',
+
+    authentication: Property.OAuth2({
+        displayName: 'Authentication',
+        description: '',
+        authUrl: "https://graph.facebook.com/oauth/authorize?",
```

Comment:
The question mark here will probably lead to issues when the actual parameters get appended
```suggestion
        authUrl: "https://graph.facebook.com/oauth/authorize",
```

Reply:
Thanks @JanHolger, changed to remove question mark